packages/http-client-java/emitter/src/code-model-builder.ts

```diff
@@ -1807,7 +1824,10 @@ export class CodeModelBuilder {
           }
         }
 
-        const schemaName = groupToRequestConditions ? "RequestConditions" : "MatchConditions";
+        let schemaName = groupToRequestConditions ? "RequestConditions" : "MatchConditions";
+        if (!this.isBranded()) {
+          schemaName = "Http" + schemaName;
```

Comment:
they have different name...
https://github.com/Azure/azure-sdk-for-java/blob/main/sdk/clientcore/core/src/main/java/io/clientcore/core/http/models/HttpRequestConditions.java